test/e2e/lib/helpers/sdk.ts

```diff
@@ -1,7 +1,8 @@
-import { getTestServers, waitForServersIdle } from '../framework'
-import { browserExecuteAsync, deleteAllCookies } from './browser'
+import { RumEvent } from '@datadog/browser-rum'
+import { EventRegistry, getTestServers, waitForServersIdle } from '../framework'
+import { browserExecute, browserExecuteAsync, deleteAllCookies } from './browser'
 
-export async function flushEvents() {
+export async function flushEvents(bridgeEvents?: EventRegistry) {
```

Comment:
Do we need all the logic about servers for bridge events?
if not, we could introduce a dedicated flush method instead of mixing the concerns

Reply:
The flush events does not focus on server or bridge events. It basically wait 200ms and wait for idle to be sure that the SDK send all events before testing. Flush event also does a page redirection to handle session replay event. I can't get bridge event after the redirection which is why I put the code inside the flush event.

Reply:
IMO, we probably don't need to wait for server idle since we don't send events to servers, but it should not hurt.